website/blog/2019-06-11-radium.md

```diff
@@ -0,0 +1,169 @@
+---
+author: Tomasz Godzik
+title: Metals v0.6.1 - Radium
+authorURL: https://twitter.com/tomekgodzik
+authorImageURL: https://avatars1.githubusercontent.com/u/3807253?s=460&v=4
+---
+
+We are excited to announce the release of Metals v0.6.1, codename "Radium" 🎉 The
+release mostly focused on easier integration with the most popular build tools
+such as Mill, Gradle and Maven.
+
+Metals is a language server for Scala that works with VS Code, Atom, Vim,
+Sublime Text and Emacs. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) along with contributors from the
+community.
+
+In this release we merged 24 PRs and closed 6 issues, full details:
+https://github.com/scalameta/metals/milestone/7?closed=1.
+
+## TL;DR
+
+- auto-import with gradle, maven and mill
+- better handling of requests that access the presentation compiler
+- code completions on keywords
+- VS code doesn't compile projects until it is focused
+- Metals is now a default server in the Sublime LSP package
+
+Check out the website and give Metals a try: https://scalameta.org/metals/
+
+## Auto-import with gradle, maven and mill
+
+Just as with `sbt` when you open a new project that contains scala code and is
+built with `Gradle`, `Maven` or `Mill`, a pop-up will show prompting you to
```

Comment:
The backticks seem unnecessary for tools names. I would use sbt, Gradle, Maven and Mill without backticks